workflows/templates/ZWESECUR.vtl

```diff
@@ -482,20 +491,25 @@ GROUP(&STCGRP.) +
 STCID(&AUXSTC.)
 F ACF2,REFRESH(STC)
 *
-* DEFINE ZOWE SERVER PERMISIONS ...................................
+* DEFINE XMEM SECURITY RESOURCES ..................................
```

Comment:
I've checked the RACF statements.  None of the changes mention XMEMUSER.  This may be OK.   Elsewhere XMEMUSER and AUXUSER are set to the same value.  I think what you're saying with these changes is that ZOWEUSER and now also AUXUSER can both access the XMEM server (ZWES.IS).

Reply:
Correct. I grant AUXUSER access to ZWES.IS, since that STC needs it. And I don't make the assumption that AUXUSER = XMEMUSER, since they may be different on some customer sites.